docusaurus/docs-internals/editor-controls/data-controls/customCss.md

```diff
@@ -0,0 +1,212 @@
+| `disabled?`        | `boolean`   |    `false`    | Configure the condition under which the control is disabled or enabled. |
+| `config?.size`     | `Size`      |  `"medium"`   | The size of the custom css component <br/> <br/> `Size = "short" \| "medium" \| "large" \| "auto"`  |                 
+| `config?.language` | `"html"`  \|  `"css"`   \| `"markdown"`  \| `"xml"`  |   `"css"`   | Specify in which language the code will be written |
+
+### Return value
+
+The return value is a `string` representing the text field value.
+
+```js
+{
+  value: "a { padding: 20px; }";
+}
+```
+
+### Usage
+
+
+#### Basic example
+Standard definition with only the required keys. This control will be displayed on all devices.
+
+```js
+{
+  id: "code", 
+  type: "codeMirror"
+}
+```
+
+#### Label example
+Adding a label on the left side of the control.
+
+```js
+{
+  id: "code", 
+  label: "Custom",
+  type: "codeMirror"
+}
+```
+
+#### Devices examples
+It will be rendered on all devices. This value can be skipped because it is set to `"all"` by default.
+
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  devices: "all"
+}
+```
+
+Rendering will occur only on `desktop`.
+
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  devices: "desktop"
+}
+```
+The display is limited to responsive modes, specifically `tablet` and `mobile`.
+
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  devices: "responsive"
+}
+```
+
+#### Disabled examples
+
+Control will be disabled. Normally, here should be your dynamic condition.
+
+```js
+{
+  id: "code", 
+  type: "codeMirror", 
+  disabled: true
+}
+```
+
+Control will be disabled when `videoType` variable will be `"custom"`.
+`getValue` is a getter function that allows us to retrieve the value of controls by their id.
+`"videoType"` is the id of the `"select"` control below.
+
+```js
+const getToolbarContols = ({ getValue }) => {
+  const videoType = getValue("videoType");
+
+  return [
+    {
+      id: "videoType",
+      type: "select",
+      choices: [
+        { title: "Youtube", value: "youtube" },
+        { title: "Custom", value: "custom" }
+      ]
+    },
+    {
+      id: "code",
+      type: "codeMirror",
+      disabled: videoType === "custom"
+    }
+  ]
+}
+```
+
+#### Config value for `size` examples
+When you add the `size` in the config, you specify the size of the option.
+
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  config: {
+    size: "short"
+  }
+}
+```
+
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  config: {
+    size: "medium"
+  }
+}
+```
+
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  config: {
+    size: "large"
+  }
+}
+```
+
+```js
+{
+  id: "code", 
+  type: "codeMirror",
+  config: {
+    size: "auto"
+  }
+}
+```
+
+#### Config value for `language` examples
+When you add the `language` in config, specify the language in which you will write the code.
```

Comment:
"In the examples below, we specify the language in which we will write the code."